src/e2e-test/features/source.control.management.sync.apps.feature

```diff
@@ -0,0 +1,32 @@
+#
+# Copyright © 2023 Cask Data, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+#
+
+@Integration_Tests
+Feature: Source Control Management - Pulling and pushing applications
+
```

Comment:
so we want to have multiple scm feature files for testing? @de-lan wdyt

Reply:
I'm fine with it. Since repo config CRUD is kinda separate ops